rules/windows/defense_evasion_parent_process_pid_spoofing.toml

```diff
@@ -0,0 +1,64 @@
+[metadata]
+creation_date = "2021/07/14"
+maturity = "production"
+updated_date = "2021/07/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies parent process spoofing used to thwart detection. Adversaries may spoof the parent process identifier (PPID)
+of a new process to evade process-monitoring defenses or to elevate privileges.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Parent Process PID Spoofing"
+references = ["https://blog.didierstevens.com/2017/03/20/"]
+risk_score = 73
+rule_id = "c88d4bd0-5649-4c52-87ea-9be59dbfbcf2"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+/* This rule is compatible with Elastic Endpoint only */
+
+sequence by host.id, user.id with maxspan=5m
+ [process where event.action == "start" and
+  process.Ext.token.integrity_level_name != "system" and
+  (
+    process.pe.original_file_name : ("winword.exe", "excel.exe", "outlook.exe", "powerpnt.exe", "eqnedt32.exe",
```

Comment:
Looks like this rule looks for very specific parents, which I think makes sense today.

In the future, once we add more parent fields like `process.parent.Ext.real.executable`, then we should be able to be more aggressive in our logic and add exceptions for the known spoofers like AppInfo.

Reply:
yep, that's the plan I have on mind (for performance reasons).